src/components/breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import {
+  BreadcrumbLink as ChakraBreadcrumbLink,
+  BreadcrumbLinkProps as ChakraBreadcrumbLinkProps,
+  Text,
+  useMultiStyleConfig,
+} from '@chakra-ui/react';
+
+interface BreadcrumbLinkProps extends ChakraBreadcrumbLinkProps {
+  href?: string;
+}
+
+const BreadcrumbLink: FC<PropsWithChildren<BreadcrumbLinkProps>> = ({
+  children,
+  ...itemProps
+}) => {
+  const styles = useMultiStyleConfig('Breadcrumbs');
+  const { href } = itemProps;
+
+  return href ? (
+    <ChakraBreadcrumbLink __css={styles.link} href={href} {...itemProps}>
+      {children}
+    </ChakraBreadcrumbLink>
+  ) : (
+    <Text color="gray.900">{children}</Text>
```

Comment:
We add the styles on theme so far. Is it possible adding the style on the corresponding theme?